marlowe-dashboard-client/package.json

```diff
@@ -3,15 +3,10 @@
   "version": "1.0.0",
   "scripts": {
     "webpack": "NODE_ENV=production NODE_OPTIONS=\"--max-old-space-size=8192\" DEBUG=purs-loader* DEBUG_DEPTH=100 webpack --progress --bail --mode=production -p",
-    "webpack:watch": "PATH=$PATH:../releases/psc-package DEBUG=purs-loader* DEBUG_DEPTH=100 webpack --progress --display-error-details --display verbose --watch",
-    "webpack:server": "webpack-dev-server --progress --inline --hot --mode=development --host 0.0.0.0 --display verbose",
-    "webpack:server:debug": "DEBUG=purs-loader* DEBUG_DEPTH=100 webpack-dev-server --progress --inline --hot",
+    "webpack:server": "webpack serve --node-env=development --progress --mode=development --liveReload",
```

Comment:
why live reload? Is HMR broken? It's broken in the website so I had to use liveReload, but I'm not sure if that is because of how I configured webpack or a v5 issue